app/interfaces/MatchTeamStatsDTO.java

```diff
@@ -0,0 +1,167 @@
+package interfaces;
+
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TeamStatsDTO contains all of the information necessary for the stats array list
+ *
+ * @author jshyo
+ *
+ */
+public class MatchTeamStatsDTO extends MatchTeamDTO{
+    // formatter
+    public static final DecimalFormat DF = new DecimalFormat( "#.##" );
+
+    // Display stats
+    private String opponent;

```

Comment:
Some other fields that should be included in this DTO are: opponentId, round, overtime tossups, the match id